tests/test_translator.py

```diff
@@ -277,6 +281,232 @@ def test_parse_raw_request_no_path(self):
         with self.assertRaises(ValueError):
             translator.parse_raw_request(raw_request)
 
+    def test_process_arguments_with_no_arguments(self):
+        # Arguments
+        args = argparse.Namespace(
+                    interactive=False,
+                    language=None,
+                    proxy=None,
+                    data=None,
+                    request=None,
+                    search_regex=None,
+                    search_string=None,
+                    file=None
+                )
+
+        with self.assertRaises(SystemExit) as exc:
+            translator.process_arguments(args)
+
+        self.assertEqual(exc.exception.code, -1)
+
+    def test_process_arguments_with_interactive_mode(self):
+        args = argparse.Namespace(
+                    interactive=True,
+                    language="bash",
+                    proxy=None,
+                    data=None,
+                    request=None,
+                    search_regex=None,
+                    search_string=None,
+                    file=None
+                )
+
+        with self.assertRaises(SystemExit) as exc:
+            with mock.patch('http_request_translator.translator.take_headers', side_effect=KeyboardInterrupt):
+                translator.process_arguments(args)
+
+        self.assertEqual(exc.exception.code, 0)
+
+    def test_process_arguments_with_request(self):
+        request_data = """GET  HTTP/1.1\nHost: google.com\nCache-Control: no-cache"""
+        args = argparse.Namespace(
+                    interactive=False,
+                    language=["bash"],
+                    proxy=None,
+                    data=None,
+                    request=request_data,
+                    search_regex=None,
+                    search_string=None,
+                    file=None
+                )
+
+        self.assertEqual(type(translator.process_arguments(args)), type({}))
+
+    def test_process_arguments_with_file(self):
+        request_data = """GET  HTTP/1.1\nHost: google.com\nCache-Control: no-cache"""
+        with open('temp', 'w+') as f:
+            f.write(request_data)
+            f.close()
```

Comment:
The `with` statement automatically closes the file so you don't have to do this manually. See: http://effbot.org/zone/python-with-statement.htm